frontend/src/components/common/SectionHeader.tsx

```diff
@@ -66,6 +59,11 @@ export default function SectionHeader(props: SectionHeaderProps) {
             )}
           </Box>
         )}
+        {!!props.subtitle && (
+          <Typography variant="h6" component="h2" sx={{ fontStyle: 'italic' }}>
```

Comment:
It looks like both title and subtitle are using h2, I'd change subtitle to 'p'. I know this was like this before but it'd be nice to change it here as well 

![image](https://github.com/user-attachments/assets/2729abb5-5923-47da-b249-eec95c402f58)

Reply:
done